runbotics/runbotics-orchestrator-ui/src/src-landing/components/BlogCard/BlogCard.tsx

```diff
@@ -50,7 +50,7 @@ const BlogCard: FC<BlogCardProps> = ({ post, className }) => {
                 <div className={styles.wrapper}>
                     {post.featuredImage?.url &&
                         <Image
-                            src={post.featuredImage?.url}
+                            src={`${post.featuredImage.url}?w=384`}
```

Comment:
are you sure the "?" symbol is unnecessary here?

Reply:
I guess so, the <Image ... /> component is displayed if url exists - condition in line 51